### PR TITLE
building the zoltan package

### PR DESCRIPTION
**Pull request by rishi9699**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/zoltan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        - mpich 3.2
+    run:
+        - mpich 3.2*
+test:
+  imports:
+    - zoltan
+
+about:
+    home: http://www.cs.sandia.gov/Zoltan/
+    license: BSD
```
Comment (scopatz) [inline recipes/zoltan/meta.yaml]: How many clauses should be listed on this line.

File: recipes/zoltan/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    license_file: LICENSE.txt
```
Comment (scopatz) [inline recipes/zoltan/meta.yaml]: Also, `license_family: BSD` should be added here

Comment (rishi9699) [reply recipes/zoltan/meta.yaml]: @scopatz  Okay, got it. 
Also, I needed a little advide from you.. Actually I am building a conda-forge package(apart from this one) and I had a doubt. My package(say A) has an optional build time dependency(say B), in a way that suppose if the user has already B installed on his machine then A will install additional features accordingly but if B is not present then A is installed just at a basic level without any problem. So is there some way I can do this with conda-forge without creating two separate packages?
Thanks

Comment (scopatz) [reply recipes/zoltan/meta.yaml]: Well you would end up with two separate packages, but you could use outputs to have these both in one feedstock/recipe: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#outputs-section

**Comment by SylvainCorlay** [comment]
Hey @rishi9699, we got zoltan to build succesfully in PR #9796 by @davidbrochart.

Do you want to be added as a maintainer to the feedstock in the other PR?

**Comment by rishi9699** [comment]
> Hey @rishi9699, we got zoltan to build succesfully in PR #9796 by @davidbrochart.
> 
> Do you want to be added as a maintainer to the feedstock in the other PR?

@SylvainCorlay Yes, do add me as a maintainer. Thanks.

**Comment by SylvainCorlay** [comment]
Closing since zoltan is now on conda-forge.

**Comment by SylvainCorlay** [comment]
@rishi9699 thanks, I missed your reply. Will add you as a maintainer when the feedstock is created.

**Pull request closed without merging**